### PR TITLE
[TD]migrate DrawingWB projection functions to TechDrawWB

### PR DESCRIPTION
**Pull request by WandererFan**
The Drawing Workbench is marked as "deprecated" since v0.17.  There are functions related to projections that exist only in Drawing.  This PR moves these functions to the TechDraw Workbench.  

Thank you for creating a pull request to contribute to FreeCAD! To ease integration, we ask you to conform to the following items. Pull requests which don't satisfy all the items below might be rejected. If you are in doubt with any of the items below, don't hesitate to ask for help in the [FreeCAD forum](https://forum.freecadweb.org/viewforum.php?f=10)!

- [ ]  Your pull request is confined strictly to a single module. That is, all the files changed by your pull request are either in `App`, `Base`, `Gui` or one of the `Mod` subfolders. If you need to make changes in several locations, make several pull requests and wait for the first one to be merged before submitting the next ones
- [ ]  In case your pull request does more than just fixing small bugs, make sure you discussed your ideas with other developers on the FreeCAD forum
- [ ]  Your branch is [rebased](https://git-scm.com/docs/git-rebase) on latest master `git pull --rebase upstream master`
- [ ]  All FreeCAD unit tests are confirmed to pass by running `./bin/FreeCAD --run-test 0`
- [ ]  All commit messages are [well-written](https://chris.beams.io/posts/git-commit/) ex: `Fixes typo in Draft Move command text`
- [ ]  Your pull request is well written and has a good description, and its title starts with the module name, ex: `Draft: Fixed typos`
- [ ]  Commit messages include `issue #<id>` or `fixes #<id>` where `<id>` is the [FreeCAD bug tracker issue number](https://freecadweb.org/wiki/tracker#GitHub_and_MantisBT) in case a particular commit solves or is related to an existing issue on the tracker. Ex: `Draft: fix typos - fixes #0004805`

And please remember to update the Wiki with the features added or changed once this PR is merged.  
**Note**: If you don't have wiki access, then please mention your contribution on the [0.20 Changelog Forum Thread](https://forum.freecadweb.org/viewtopic.php?f=10&t=56135).

---

### CODE REVIEW

File: src/Mod/TechDraw/TDTest/TDPyTest.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
```
Comment (benj5378) [inline src/Mod/TechDraw/TDTest/TDPyTest.py:3]: This does not have anything to do with the moving, but I believe these lines aren't needed when it's run through FreeCAD. From some very quick searches online, they are used to make sure that they are interpreted by Python if opened isolated on Linux.

It was just a very quick serach, but I don't think they are needed

File: src/Mod/TechDraw/TDTest/TDPyTest.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+# basic test script for TechDraw Py functions migrated from Drawing
+
+from __future__ import print_function
+
+import FreeCAD
+import Part
+import Measure
```
Comment (benj5378) [inline src/Mod/TechDraw/TDTest/TDPyTest.py:10]: Doesn't have anything to do with the moving either, but some unused modules could be removed.

```suggestion
# import Part
```

File: src/Mod/TechDraw/App/ProjectionAlgos.cpp [hunk 3]
```diff
@@ -0,0 +1,290 @@
+/***************************************************************************
+ *   Copyright (c) Jürgen Riegel          (juergen.riegel@web.de) 2002     *
```
Comment (luzpaz) [inline src/Mod/TechDraw/App/ProjectionAlgos.cpp:2]: Fix heading formatting (should look like [this](https://github.com/FreeCAD/FreeCAD/pull/5457/files#diff-06fc5002222d6a7ea53e9ddb9a565cf709e7ea9ec62841246412d436f05a6edbR2))

File: src/Mod/TechDraw/App/ProjectionAlgos.h [hunk 4]
```diff
@@ -0,0 +1,86 @@
+/***************************************************************************
+ *   Copyright (c) Jürgen Riegel          (juergen.riegel@web.de) 2009     *
```
Comment (luzpaz) [inline src/Mod/TechDraw/App/ProjectionAlgos.h:2]: Fix heading formatting (should look like [this](https://github.com/FreeCAD/FreeCAD/pull/5457/files#diff-06fc5002222d6a7ea53e9ddb9a565cf709e7ea9ec62841246412d436f05a6edbR2))

**Review by luzpaz — CHANGES_REQUESTED** [review]

**Comment by freecadci** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_5457">pipeline status</a> for feature branch [PR_5457](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_5457). Pipeline [459054346](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/459054346) was triggered at [65a2811](https://github.com/FreeCAD/FreeCAD/pull/5457/commits/65a281186dda127e27d81147c7d43d2d79e43ca3). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

File: src/Mod/TechDraw/App/ProjectionAlgos.cpp (hunk 3 above)
Comment (WandererFan) [reply src/Mod/TechDraw/App/ProjectionAlgos.cpp:2]: Why is this important?  The copyright notice has been formatted this way for 20 years.  

Not trying to be obnoxious, just don't understand.

File: src/Mod/TechDraw/TDTest/TDPyTest.py (hunk 1 above)
Comment (WandererFan) [reply src/Mod/TechDraw/TDTest/TDPyTest.py:3]: do these lines hurt anything?

Comment (benj5378) [reply src/Mod/TechDraw/TDTest/TDPyTest.py:3]: Not at all

**Pull request merged**